### ISSUE TITLE
Intermittent issue with global appsvcs lock files

### ISSUE THREAD

**Issue by oakh1** [F5Networks/f5-appsvcs-extension#406]
### Environment
 * Application Services Version: 3.24.0
 * BIG-IP Version: 14.1

### Summary
Deploying a declaration caused internal error 500 from AS3 with message saying that the global ___appsvcs_lock file wasn't found.

### Steps To Reproduce
During deployment I did not log the declaration (there are automatic deployments running and I don't have the exact version of it), but it was nothing wrong with it as it worked the other 80/100 times (apart from when it times out). Hopefully just by the error message you should be able to determine in which scenarios this could happen as I'm not able to reproduce it easily.

2. Observe the following error response:
```json
{
   "code": 500
   "message": "DELETE http://admin:XXXXXX@localhost:8100/mgmt/tm/ltm/data-group/internal/~Common~appsvcs~____appsvcs_lock Attempting to release global lock response=404 body={\"code":404,\"message\":\"01020036:3: The requested value list (/Common/appsvcs/____appsvcs_lock) was not found.\",\"errorStack\":[],\"apiError\":3}"
}
```

### Expected Behavior
The deployment to be successful

### Actual Behavior
The deployment fails

**Comment by dstokesf5** [comment]
We have seen this message before, and we have audited the code base looking for possible race conditions. Unfortunately, the error message alone has not been enough to determine why this occasionally happens. I have added this bug report to our internal product backlog as AUTOTOOL-2202. However, we are unlikely to make progress on this until some new piece of information sheds some more light on this mystery.

**Comment by dstokesf5** [comment]
Either we missed something last time or I am getting my issues confused. A developer spotted an issue where we could release the lock twice that we have now resolved. The fix should be available in the next release.

**Comment by oakh1** [comment]
Excellent. Well done!

**Comment by dstokesf5** [comment]
This issue was resolved in the 3.27.0 release.
